doc/chap0.txt

```diff
@@ -34,9 +34,13 @@
   
   -------------------------------------------------------
   [1XAcknowledgements[101X
-  [33X[0;0YI  thank  my  PhD  supervisor  Professor Jeremy Rickard, on whom I inflicted
-  multiple  early  iterations of [5XSBStrips[105X, for his time and his comments. This
-  package was much worse before his feedback.[133X
+  [33X[0;0YI  thank my PhD supervisor Prof Jeremy Rickard, on whom I inflicted multiple
```

Comment:
Just a quick note: in general I would advise against putting generated files (like these .txt, .html, .css, ... files) into a source repository. This is a common "best practice" and e.g reduces gratuitous diffs caused by changes to the GAPDoc version used by different contributors

Reply:
Thank you very much for the pointer! I will make a .gitignore file to this effect 🙂